app/content/team/users/[user]/index.js

```diff
@@ -319,8 +319,17 @@ function Index() {
               ]);
               navigation.goBack();
             } catch (e) {
-              console.log("Error removing user:", e);
-              showDialog("Error", getErrorString(e));
+              if (e.code === "storage/object-not-found") {
```

Comment:
looks like you were trying to have a temp fix here, might be better to get to the root cause so we can validate it is working for the long run